tests/container_disk_test.go

```diff
@@ -81,7 +82,10 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting and stopping the same VirtualMachineInstance", func() {
 		Context("with ephemeral registry disk", func() {
 			It("[test_id:1463][Conformance] should success multiple times", func() {
-				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+				userData := "#!/bin/bash\necho 'hello'\n"
+				vmi := libvmi.NewCirros(
+					libvmi.WithCloudInitNoCloudEncodedUserData(userData),
```

Comment:
this is not needed. Cirros factory already use this userData
https://github.com/kubevirt/kubevirt/blob/afbeb269840e984b6c49d984ed1e53ab6eb02302/tests/libvmifact/factory.go#L51-L62

Reply:
Yes, but `NewRandomVMIWithEphemeralDiskAndUserdata` also includes
```
libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
libvmi.WithNetwork(v1.DefaultPodNetwork()),
```
but for these test purposes, I think we don't need them.
So, IMO, `vmi := libvmifact.NewCirros()` should be enough!